### PR TITLE
Add Metamask Support

### PR DESCRIPTION
**Pull request by schnetzlerjoe**
This PR adds Metamask support to Akash Console allowing MM signing and deployment on Akash.

### CODE REVIEW

File: apps/deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx [hunk 1]
```diff
@@ -13,6 +13,16 @@ import { akash, akashSandbox, akashTestnet, assetLists } from "@src/chains";
 import { useSelectedNetwork } from "@src/hooks/useSelectedNetwork";
 import { customRegistry } from "@src/utils/customRegistry";
 
+import "@interchain-ui/react/styles";
+import "@interchain-ui/react/globalStyles";
```
Comment (baktun14) [inline apps/deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx]: This is already imported at the top of the file

File: apps/deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx [hunk 2]
```diff
@@ -13,6 +13,16 @@ import { akash, akashSandbox, akashTestnet, assetLists } from "@src/chains";
...
+
+let mm: any[] = [];
+if (typeof window !== 'undefined') {
+  import("@cosmos-kit/cosmos-extension-metamask").then((module) => {
+    mm = module.wallets;
+  });
+}
```
Comment (baktun14) [inline apps/deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx:21]: I think a cleaner way would be to use next dynamic import, what do you think?
```
const metamask = dynamic(() => import("@cosmos-kit/cosmos-extension-metamask").wallets, {
  ssr: false
});
```
(I haven't tested this code)

File: package.json
Comment (Redm4x) [inline package.json:1]: Some things have been removed and patch-package added, are those changes needed or a rebase issue?

File: apps/deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx (hunk 2 above)
Comment (schnetzlerjoe) [reply apps/deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx:21]: @baktun14 Tried that but it fails. Could not get it to work . This solution is definitely more stable @baktun14

File: package.json
Comment (schnetzlerjoe) [reply package.json:1]: @Redm4x Nope! I fixed it

File: apps/deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx (hunk 1 above)
Comment (schnetzlerjoe) [reply apps/deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx]: Fixed! @baktun14

**Comment by baktun14** [comment]
There are way too many changes to the package-lock.json and the package needs to be installed in the console app. I'll make a pr with the changes...

**Pull request closed without merging**